schemas/mw-article-edit.js

```diff
@@ -0,0 +1,68 @@
+{
+    "title": "MediaWiki Article Edit",
+    "description": "Represents a MW Article Edit event",
+    "type": "object",
+    "properties": {
```

Comment:
Here are the fields I'd add:  user_id, user_text, byte_length/char_length, sha1, minor